gatling-load-test/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("io.gatling.gradle") version "3.10.5"
+}
+
+configurations {
+    compileOnly {
+        extendsFrom(configurations.annotationProcessor.get())
+    }
+}
+
+dependencies {
+    implementation("io.gatling:gatling-app:${Versions.GATLING_APP}")
```

Comment:
add charts library to get reports in the form of diagrams
https://mvnrepository.com/artifact/io.gatling.highcharts/gatling-charts-highcharts/3.10.5